src/table/components/withSelections.jsx

```diff
@@ -5,9 +5,9 @@ import { getSelectionStyle } from '../utils/styling-utils';
 
 export default function withSelections(CellComponent) {
   const HOC = (props) => {
-    const { selectionState, cell, selDispatch, styling, announce } = props;
+    const { selectionstate: selectionState, cell, selectiondispatch: selectionDispatch, styling, announce } = props;
```

Comment:
what, react complains when you use camel case??

Reply:
Not here, react complains there https://github.com/qlik-oss/sn-table/pull/430/files#diff-0d7173fd8da1e342bacca6f203bd521b0300c9ab25684eb6817b7e3d4be57a0cL90

Reply:
Is there a good reason for it? I just think that it makes the variable names harder to read. What is the name of the rule?

Reply:
<img width="869" alt="Screenshot 2022-01-28 at 12 07 13" src="https://user-images.githubusercontent.com/4471489/151536589-85db131e-5d53-4574-9e9e-f1d7854b6029.png">
https://reactjs.org/warnings/unknown-prop.html The fouth point. It is due to how react is working

Reply:
https://reactjs.org/warnings/unknown-prop.html

Reply:
the second point.
<img width="875" alt="Screenshot 2022-01-28 at 12 14 33" src="https://user-images.githubusercontent.com/4471489/151537564-261c669b-5879-44a4-9ef5-9ea2fe123d71.png">
 https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes
 You are right, I will do the update.

Reply:
no, i do not find a better way.  `data-selection-state `is not converted to `selectionState`
<img width="517" alt="Screenshot 2022-01-28 at 12 18 24" src="https://user-images.githubusercontent.com/4471489/151538955-071534e0-9849-47bb-9149-74f36f3e21a3.png">

Reply:
oh ok I think i understand now. CellRenderer isn't a real react component, but a standard html component. Then I guess it should be lower case